src/wordnet_autotranslate/models/xml_synset_parser.py

```
@@ -9,6 +9,13 @@
 import logging
 from dataclasses import dataclass
 
+# POS normalization utilities (Serbian <-> English)
+try:
+    from ..utils.language_utils import LanguageUtils
+except Exception:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)